Makefile

```diff
@@ -5,6 +5,7 @@ DOCKER_COMPOSE ?= docker-compose
 OFFICIAL ?= no
 NODE ?= node
 NPM_BIN ?= npm
+KIND_BIN ?= $(shell which kind)
```

Comment:
This adds overhead to running any make target, because it has to run this shell command. I also cannot understand the utility of doing this much more complicated thing instead of just typing `kind` in the `make kind-dev-load` where it is intended to be used.